service/internal/auth/casbin.go

```diff
@@ -72,6 +73,7 @@ p,	role:admin,		/kas/v2/rewrap,						  				        *,      allow
 
 ## Role: Standard
 ## gRPC routes
+p,  role:standard,		policy.unsafe.*,															*,			deny
 p,	role:standard,		policy.*,																read,			allow
 p,	role:standard,		kasregistry.*,													read,			allow
```

Comment:
```suggestion
p,  role:standard,		policy.unsafe.*,													 *,			deny
p,	role:standard,		policy.*,																read,			allow
p,	role:standard,		kasregistry.*,													read,			allow
```

Reply:
[`cfcf791` (#1030)](https://github.com/opentdf/platform/pull/1030/commits/cfcf7918ca091535f33fe592bb94599a2aaca7a1) 

I'm actually not 100% sure casbin policy is enforced sequentially as this expects. I will evaluate separately when enabling the unsafe service.